### PR TITLE
Add fliplr and flipud numpy ops

### PR DESCRIPTION
**Pull request by rstar327**
## Summary

Add `keras.ops.fliplr` and `keras.ops.flipud` numpy operations with support across all backends (NumPy, JAX, TensorFlow, PyTorch, OpenVINO).

- `fliplr` reverses elements along axis 1 (left/right flip)
- `flipud` reverses elements along axis 0 (up/down flip)

These are standard NumPy operations commonly used in image preprocessing and data augmentation.

## Implementation

- Operation classes with `compute_output_spec` and dimension validation
- Backend implementations using native ops (`np.fliplr`, `jnp.fliplr`, `tf.reverse`, `torch.fliplr`, OpenVINO `flip`)
- Symbolic shape tests (dynamic and static) and correctness tests

### CODE REVIEW

File: keras/src/backend/jax/numpy.py [hunk 1]
```diff
@@ -748,6 +748,14 @@ def flip(x, axis=None):
     return jnp.flip(x, axis=axis)
 
 
+def fliplr(x):
+    return jnp.fliplr(x)
+
+
+def flipud(x):
+    return jnp.flipud(x)
```
Comment (hertschuh) [inline keras/src/backend/jax/numpy.py:758]: Please add `x = convert_to_tensor(x)` to both.

**Comment by rstar327** [comment]
Done
